logic/src/commonTest/kotlin/com/wire/kalium/logic/data/message/MessageRepositoryTest.kt

```diff
@@ -521,6 +521,38 @@ class MessageRepositoryTest {
         )
     }
 
+    @Test
+    fun givenSearchedMessages_whenMessageIsSelected_thenReturnMessagePosition() = runTest {
+        // given
+        val qualifiedIdEntity = TEST_QUALIFIED_ID_ENTITY
+        val conversationId = TEST_CONVERSATION_ID
+        val message = TEST_MESSAGE_ENTITY.copy(
+            id = "msg1",
+            conversationId = qualifiedIdEntity,
+            content = MessageEntityContent.Text("message 1")
+        )
+        val expectedResult = 113
```

Comment:
Suggestion: Maybe here we can rename it to have something like `index` or Position in the name ? 
`expectedMessagePosition` or `expectedResultPosition`